hazelcast/src/main/java/com/hazelcast/instance/EndpointQualifier.java

```diff
@@ -136,6 +136,26 @@ public String toString() {
                 + '\'' + '}';
     }
 
+    /**
+     * @return resolved endpoint qualifier when it is passed from the user via configuration
+     */
+    public static EndpointQualifier resolveForConfig(ProtocolType protocolType, String identifier) {
+        switch (protocolType) {
```

Comment:
Minor: Not a big deal, but to decrease the code duplication, maybe we could rewrite this as something like 
```java
if (ProtocolType.CLIENT.equals(protocolType)) {
    return CLIENT;
}
return resolve(protocolType, identifier);
```